packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -587,6 +591,111 @@ it('should not show sort icon if disabled in column level', () => {
   expect(container.querySelector('.iui-sort .iui-icon-wrapper')).toBeFalsy();
 });
 
+it('should display correct sort icons for ascending first', async () => {
+  const mockedColumns = [
+    {
+      Header: 'Header name',
+      columns: [
+        {
+          id: 'name',
+          Header: 'Name',
+          accessor: 'name',
+        },
+      ],
+    },
+  ];
+  const { container } = renderComponent({
+    columns: mockedColumns,
+    isSortable: true,
+  });
+  const {
+    container: { firstChild: sortUpIcon },
+  } = render(<SvgSortUp className='iui-icon iui-sort' aria-hidden />);
+  const {
+    container: { firstChild: sortDownIcon },
+  } = render(<SvgSortDown className='iui-icon iui-sort' aria-hidden />);
+  const nameHeader = container.querySelector(
+    '.iui-table-header .iui-cell',
+  ) as HTMLDivElement;
+  expect(nameHeader).toBeTruthy();
```

Comment:
Don't think this would matter, but this check might be unnecessary here. Since this test is not doing anything with the nameHeader. The other two existing tests that do this `nameHeader` check also use `nameHeader`, which makes the check important.

But this change is optional, just something I noticed

Reply:
I left this check in because I click on the nameHeader in lines 628, 634, and 640.